stonesoup/models/measurement/linear.py

```diff
@@ -73,7 +74,8 @@ def function(self, state_vector, noise=None, **kwargs):
         """
 
         if noise is None:
-            noise = self.rvs()
+            # noise = sp.zeros(state_vector.shape)
+            noise = self.rvs()  # TODO noise=None generates noise!
```

Comment:
As per other comment, this can be done in separate pull request.

Reply:
Done.